pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -896,26 +902,49 @@ public void failed(Throwable throwable) {
     @Override
     public List<Message<byte[]>> peekMessages(String topic, String subName, int numMessages)
             throws PulsarAdminException {
-        return sync(() -> peekMessagesAsync(topic, subName, numMessages));
+        return sync(() -> peekMessagesAsync(topic, subName, numMessages,
+                false, SubscriptionIsolationLevel.READ_COMMITTED));
+    }
```

Comment:
It's better to remove this implementation, just add a default implementation to the interface.